tests/test_server_ops.py

```diff
@@ -228,3 +228,43 @@ async def test(pgdata_path):
                 self.loop.run_until_complete(test(td))
             finally:
                 cluster.stop()
+
+    def test_server_ops_postgres_multitenant(self):
+        async def test(pgdata_path, tenant):
+            async with tb.start_edgedb_server(
+                auto_shutdown=True,
+                tenant_id=tenant,
+                reset_auth=True,
+                postgres_dsn=f'postgres:///?user=postgres&host={pgdata_path}',
+                runstate_dir=None if devmode.is_in_dev_mode() else pgdata_path,
+            ) as sd:
+                con = await sd.connect()
+                try:
+                    await con.execute(f'CREATE DATABASE {tenant}')
+                    await con.execute(f'CREATE SUPERUSER ROLE {tenant}')
+                    databases = await con.query('SELECT sys::Database.name')
+                    self.assertEqual(set(databases), {'edgedb', tenant})
+                    roles = await con.query('SELECT sys::Role.name')
+                    self.assertEqual(set(roles), {'edgedb', tenant})
+                finally:
+                    await con.aclose()
+
+        with tempfile.TemporaryDirectory() as td:
+            cluster = pgcluster.get_local_pg_cluster(td)
+            cluster.set_connection_params(
+                pgconnparams.ConnectionParameters(
+                    user='postgres',
+                    database='template1',
+                ),
+            )
+            self.assertTrue(cluster.ensure_initialized())
+            cluster.start()
+            try:
+                self.loop.run_until_complete(
+                    asyncio.wait([
```

Comment:
`asyncio.wait()` doesn't propagate the errors, please use e.g. `taskgroup` instead.

Reply:
Good point.  Fixed.